src/webgpu/api/validation/encoding/encoder_open_state.spec.ts

```diff
@@ -298,6 +301,11 @@ g.test('render_pass_commands')
     const encoder = t.device.createCommandEncoder();
     const renderPass = beginRenderPassWithQuerySet(t, encoder, querySet);
 
+    t.skipIf(
+      command === 'setImmediates' && !('setImmediates' in renderPass),
+      'setImmediates not supported'
+    );
```

Comment:
nit: prefer to do this in `beforeAllSubcases` - just as a matter of style, it's not a big deal for this test since it only has 3 subcases

Reply:
done